cmd/describe/cluster/cmd.go

```diff
@@ -133,6 +134,19 @@ func run(_ *cobra.Command, argv []string) {
 		reporter.Errorf("Failed to parse creator ARN for cluster '%s'", clusterKey)
 		os.Exit(1)
 	}
+	phase:= ""
+	if cluster.State() ==  cmv1.ClusterStatePending{
+		phase = "Preparing account"
+	}
+
+	if cluster.State() == cmv1.ClusterStateInstalling {
+		if !cluster.Status().DNSReady() {
```

Comment:
Is there a phase for when the DNS _is_ ready? Or will that just show the state without the phase?

Reply:
@vkareh it will just show the status installing.

Reply:
nice, thanks